bambinomio/assets/html/order.html

```diff
@@ -126,7 +126,7 @@ <h2>Piegāde</h2>
                     <!-- placeholder -->
                 </fieldset>
                 <div id="orderConfirm" class="w3-container w3-hide">
-                    <button type="submit" onclick="emailOrder();"
+                    <button type="submit" onclick="emailOrder(); oneclick=confirmation()"
```

Comment:
Iespējams, ka šitā būtu ok: 
<button type="submit" onclick="emailOrder(); confirmation()"